postgresql/connection_test.go

```diff
@@ -83,7 +88,7 @@ func TestParseConnectionURL(t *testing.T) {
 	var s string
 	var err error
 
-	s = "postgres://anakin:skywalker@localhost/jedis"
+	s = "postgres://anakin:skywalker@localhost:1234/jedis"
```

Comment:
I think we should keep this line to test the case with no port defined, can you leave it like that and add another test for the port case?